.github/workflows/ci-weekly.yml

```diff
@@ -1,74 +1,165 @@
-name: CI - FAST Validation
+name: CI - FAST
 
 on:
-  schedule:
-  - cron: 0 7 * * 3
-
+  push:
+    branches:
+    - windows-fails-ci-validation
```

Comment:
Does this branch need to persist beyond this initial work?

Reply:
persistence is not really useful

Reply:
I just remove windows-fails-ci-validation branch so we can keep the master clean